openedx/core/djangoapps/waffle_utils/__init__.py

```diff
@@ -6,7 +6,7 @@
 Usage:
 
 For Waffle Flags, first set up the namespace, and then create flags using the
-namespace.  For example:
+namespace.  For example::
```

Comment:
What's with the double colons?

Reply:
Python docstrings are RST, so these make the following examples code-blocks.